### ISSUE TITLE
Panning doesn't release mouse in firefox when map is inside a disabled fieldset

### ISSUE THREAD

**Issue by olivermay** [Leaflet/Leaflet#5848]
## How to reproduce

- Leaflet version I'm using: 1.2.0 (older versions are the same, also tested 0.7.7)
- Browser (with version) I'm using: Firefox 54.0 64 bit (also on 55, 56 on Windows)
- OS/Platform (with version) I'm using: Ubuntu Linux 16.04
- Tested in other Browsers/platforms: The problem does **not** occur in chrome/linux, chrome/windows, edge/windows

1. Place a leaflet map inside a HTML fieldset and set the disabled property.
2. Use the mouse to start panning

## What behaviour I'm expecting and which behaviour I'm seeing
I expect normal panning behaviour.
When I am panning and release the mouse button, I am still panning. 

## Minimal example reproducing the issue

https://plnkr.co/edit/bU5RxPt11kWJqEgh1nKg

**Comment by simon1tan** [comment]
It looks to affect Firefox. It works fine in Chrome.

**Comment by jguthrie100** [comment]
Should the map be entirely disabled if its in a disabled fieldset?

**Comment by olivermay** [comment]
I don't think so, the behaviour in Chrome is what I was expecting: the disabled fieldset has no impact on the map.

**Comment by Falke-Design** [comment]
Panning is not working anymore in Firefox. In Chrome is still all working like expected

**Comment by johnd0e** [comment]
Updated example with master leaflet https://plnkr.co/edit/KHtmZEPi6BOltI0K
